modules/ROOT/pages/install-aws.adoc

```diff
@@ -190,27 +199,24 @@ terraform apply \
 If you experience an error related to AWS authorization, ensure you're using the same terminal window for 
 verifying the variables and running the Terraform command.
 
-. Run the script.
-
+. Run the script using Docker.
++
 The Terraform script provisions the infrastructure and runs the installation script on each VM. When complete, 
 the Runtime Fabric update to Active status in Runtime Manager.
-
++
 [NOTE]
 This step installs Runtime Fabric across all servers to form a cluster. It may take 15-25 minutes or longer to complete.
 
-
-==== Run using Docker
-
-. Navigate to the `../rtf-install-scripts/` directory. You must run terraform from this directory. When running the `dir` or `ls` command, you should see the `aws` directory listed along with other directories (`azure`, `manual`, etc).
-. Initialize terraform (this only has to be performed once):
+.. Navigate to the `../rtf-install-scripts/` directory. You must run terraform from this directory. When running the `dir` or `ls` command, you should see the `aws` directory listed along with other directories (`azure`, `manual`, etc).
```

Comment:
GLOBAL
Also, see if there is a way to promote this step and its peers, to avoid nesting.
```suggestion
.. Navigate to the `../rtf-install-scripts/` directory. 
You must run Terraform from this directory. When running the `dir` or `ls` command, you should see the `aws` directory listed along with other directories (`azure`, `manual`, and so on).
```